charts/nexus-iq/README.md

```diff
@@ -90,6 +93,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `iq.readinessProbe.failureThreshold`           | Number of attempts before failure                                                                                                                                                       | 3                                                           |
 | `iq.readinessProbe.timeoutSeconds`             | Time in seconds after readiness probe times out                                                                                                                                         | 2                                                           |
 | `iq.readinessProbe.successThreshold`           | Number of attempts for the probe to be considered successful                                                                                                                            | 1                                                           |
+| `initContainers.enabled`                       | Change file ownership during IQ upgrade from older version                                                                                                                              | false                                                       |
```

Comment:
I'd name this something more specific to the "chown for upgrades". "initContainers" is the technique, not the goal.
"fixOwnership.enabled"? Is there a time we'd not want this enabled?

Reply:
Sure, fixOwner looks good. Will change and update.
This setting would be disabled by default and only needs when upgrading from older IQ versions <117 or <100

https://help.sonatype.com/iqserver/installing/upgrading-the-iq-server#UpgradingtheIQServer-UpgradingfromRelease117orEarliertoRelease118orLater